bin/mocha-chrome

```diff
@@ -62,9 +66,16 @@ const mochaDefaults = {
   useColors: !flags.colors
 };
 const mocha = deepAssign(mochaDefaults, JSON.parse(flags.mocha || '{}'));
-const logLevel = flags.logLevel || 'error';
-const ignoreResourceErrors = flags.ignoreResourceErrors || false
-const options = { url, logLevel, mocha, ignoreResourceErrors };
+const chromeFlags = JSON.parse(flags.chromeFlags || '[]');
+const options = {
```

Comment:
this is good as is, there's nothing functionally wrong here. just a heads up that after this is merged I'm going to be switching this to the defaults pattern that you can see on line 64 and 68. it's your call if you want to do that here or if you'd like to merge as-is and have me take care of that.

Reply:
I removed the rest of defaults, as they are already set in `index.js` file.